integration_tests/features/BlockExplorerGRPC.feature

```diff
@@ -0,0 +1,12 @@
+@block-explorer
+Feature: Block Explorer GRPC
+
+  @critical
+  Scenario: GetNetworkDifficulty
+    Given I have a seed node NODE
+    And I have wallet WALLET connected to all seed nodes
+    And I have a merge mining proxy PROXY connected to NODE and WALLET
+    When I merge mine 2 blocks via PROXY
+    Then all nodes are at height 2
+    When I request the difficulties of a node NODE
+    Then Difficulties are available
```

Comment:
```suggestion
    Then the last network difficulty is 1
```

Reply:
Hmm, while the values of only the first block is checked (reverse ordered so it at the end) as we know it will always be the same, it does also check that there are more than just that value in the array. 

Would you still like this changed?